### PR TITLE
Stop pushing to old ACRs

### PR DESCRIPTION
**Pull request by arroyc**
as mentioned in the title

### CODE REVIEW

File: vsts/pipelines/templates/_releaseBaseImagesStepTemplate.yml [hunk 1]
```diff
@@ -29,43 +29,13 @@ steps:
...
 
-- task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
-  displayName: 'Pull and create release tags for staging ACR'
-  inputs:
-    type: FilePath
-    scriptPath: ./vsts/scripts/tagBaseImagesForRelease.sh
-    args: '${{ parameters.baseImageName }} ${{ parameters.acrProdName }}'
-
 - task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
   displayName: 'Pull and create release tags for PME staging ACR'
```
Comment (qianz2) [inline vsts/pipelines/templates/_releaseBaseImagesStepTemplate.yml:33]: so are we now only push to pme ACRs and will remove all non-pme ACRs?

Comment (qianz2) [reply vsts/pipelines/templates/_releaseBaseImagesStepTemplate.yml:33]: please make sure those build/runtime images under old ACRs will not be used in the future

**Review by qianz2 — COMMENTED** [review]
Could you also trigger a nightly build afterwards, since it involved in some script changes. btw, looks that the baseImages-ruby pipelines automated triggered, I'll try to make it stop :)

File: vsts/pipelines/templates/_releaseBaseImagesStepTemplate.yml (hunk 1 above)
Comment (arroyc) [reply vsts/pipelines/templates/_releaseBaseImagesStepTemplate.yml:33]: we dont use them anymore ... it was just sitting there for no reason

**Comment by arroyc** [comment]
> Could you also trigger a nightly build afterwards, since it involved in some script changes. btw, looks that the baseImages-ruby pipelines automated triggered, I'll try to make it stop :)

i can surely trigger .. but these scripts won't do much in nightly build though .. as they are only used during ci build (pulling and tagging etc). I can trigger base image builds tough ..

**Review by qianz2 — APPROVED** [review]

**Pull request merged**